src/official-account/official-account.ts

```diff
@@ -252,6 +254,40 @@ class OfficialAccount extends EventEmitter {
     return ret.body
   }
 
+  async uploadTempFile (args: {
+    file: FileBox,
+    touser: string,
+    msgtype: OAMessageType
+  }): Promise<ErrorPayload> {
+    log.verbose('OfficialAccount', 'uploadTempFile(%s)', JSON.stringify(args))
+    const msgSet: Set<string> = new Set(['image', 'video', 'voice'])
```

Comment:
Could we use the typing system to make sure the `args.msgtype` is valid instead of checking a `Set` at here?

That will get rid of the unnecessary additional payload for our program, and also make the code more readable.

Reply:
It's amazing that the choice typing of parameters is so simple to be implemented. Get it ~